docs/release-notes/december-2023.md

```diff
@@ -0,0 +1,116 @@
+---
+slug: december-2023
+append_help_link: true
+hide_title: true
+description: >-
+    Release notes include the changes, fixes, and additions in specific versions of Semgrep.
+toc_max_heading_level: 3
+title: December 2023
+tags:
+  - Release notes
+---
+
+# December 2023 release notes
+
+## 🔧 OSS Engine
+
+* The following versions of the OSS Engine were released in December 2023:
+
+  * [<i class="fas fa-external-link fa-xs"></i>
+    1.52.0](https://github.com/returntocorp/semgrep/releases/tag/v1.52.0)
+  * [<i class="fas fa-external-link fa-xs"></i>
+    1.53.0](https://github.com/returntocorp/semgrep/releases/tag/v1.53.0)
+  * [<i class="fas fa-external-link fa-xs"></i>
+    1.54.0](https://github.com/returntocorp/semgrep/releases/tag/v1.54.0)
+  * [<i class="fas fa-external-link fa-xs"></i>
+    1.54.1](https://github.com/returntocorp/semgrep/releases/tag/v1.54.1)
+
+## 🌐 Cloud Platform
+
+### Added
+
+* Semgrep IDE integrations now cache information about the repository so that it
+  does not traverse the entire repository on every scan to determine if the
+  files are valid targets for scanning; this improves scan times.
+* Users can now ignore findings locally in Semgrep IDE Extensions. These changes
+  persist between restarts.
+* Added authentication provider information for each user under **Settings** >
```

Comment:
This was already there. We just moved the column (we only added it to the RBAC teams page which is still in private beta).